src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/RoboticsDataMapper.java

```diff
@@ -42,6 +42,11 @@
     public static final String APPLICANT_ID = "001";
     public static final String RESPONDENT_ID = "002";
 
+    public static final String APPLICANT2_SOLICITOR_ID = "003";
+    public static final String RESPONDENT2_SOLICITOR_ID = "004";
+    public static final String APPLICANT2_ID = "003";
+    public static final String RESPONDENT2_ID = "004";
```

Comment:
@jeejavr not sure if this is relevant, do these values need to be unique?

Reply:
@HarryH96 not sure about that as there was a previous implementation for applicant 1 and solicitor 1. I was just updating it for applicant 2 and solicitor 2. Please let me know your thoughts on alternate implementations.

Reply:
@jeejavr  **buildSolicitors** method needs updating otherwise in MP scenario it will never build the whole solicitors list. However this change is now out of scope of this ticket as we are not attaching the json anymore to the email but would be required in one of the RPA stories we are going to look at.

Reply:
@HarryH96 These Ids are only used in the json to represent a unique object. So for example a list of solicitors will be populated within the json and each solicitor object will have its own unique Id.  See this RPA json to example to see how these ids are used - https://jsonformatter.org/60893c.